packages/ozone/src/tag-service/embed-tagger.ts

```diff
@@ -15,24 +21,29 @@ export class EmbedContentTypeTagger extends ContentTagger {
     )
   }
 
-  async getTags(): Promise<string[]> {
+  async buildTags(): Promise<string[]> {
     try {
-      if (!this.isApplicable()) {
-        return []
-      }
       const recordValue = await this.getRecordValue()
       if (!recordValue) {
         return []
       }
       const tags: string[] = []
       if (
-        AppBskyEmbedImages.isView(recordValue) &&
-        (AppBskyEmbedImages.isMain(recordValue.embed) ||
-          AppBskyEmbedRecordWithMedia.isMain(recordValue.embed))
+        AppBskyFeedPost.isRecord(recordValue) &&
+        AppBskyEmbedRecordWithMedia.isMain(recordValue.embed)
```

Comment:
There are four cases for `recordValue.embed` we should be looking for:
 - image embed
 - video embed
 - "record with media" containing image media
 - "record with media" containing video media